lib/targzip/targz.go

```diff
@@ -171,6 +171,19 @@ func (t *TarGzip) Name() string {
 	return t.fileName
 }
 
+func (t *TarGzip) Size() int64 {
```

Comment:
You can just use `fi, err := os.Stat(t.Name())` instead of opening and closing the file.